styles/tokens/alert.json

```diff
@@ -198,7 +198,7 @@
         },
         "fail": {
           "comment": "The icon color of the Fail Alert component.",
-          "value": "{pink.value}",
+          "value": "{orange.value}",
```

Comment:
verify we mentioned the fail alert color change in release notes

Reply:
Checked. It's in there.